data_structures/src/superblock.rs

```diff
@@ -257,11 +264,45 @@ impl SuperBlockState {
         self.received_superblocks.contains(sbv)
     }
 
-    /// Updates the current superblock signing committee for a given superblock index
-    /// #FIXME This function should be update with issue 1395
-    pub fn update_superblock_signing_committee(&mut self, _current_index: u32) {
-        self.current_signing_committee = self.previous_ars_identities.clone();
+    /// Updates the current superblock signing committee for a given superblock hash
+    pub fn update_superblock_signing_committee(
+        &mut self,
+        _signing_committee_size: u32,
+    ) -> Option<HashSet<PublicKeyHash>> {
```

Comment:
Internally updating self.current_signing_committee and also returning it seems a bit odd to me, although I understand this is for testing purposes. Maybe we can just make this agnostic of superblock_state and make all the necessary parameters be inserted as arguments? We could just call this function something like calculate_superblock_signing_committee